src/VCAuthn/appsettings.json

```diff
@@ -15,5 +15,20 @@
     "CertificateFilename": "secret.rsa",
     "PublicOrigin": "http://localhost:5000",
     "AccessTokenLifeTimeSeconds": 172800
+  },
+  "ACAPY": {
```

Comment:
Would it be possible to rename ACAPY to AriesCloudAgent or are we coupled for the bcgov impl?

Reply:
It is possible, but as far as i know ACAPy was a requirement for this implementation, so I think stick to this name might be a fair idea.